.github/workflows/ci.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-22.04 ]
+        os: [ ubuntu-24.04 ]
```

Comment:
This does not really affect the workflow since CI is running in docker container but updated accordingly.